File: task-03/JavaScript/SubTask3.js
const readline = require('readline').createInterface({
    input: process.stdin,
    output: process.stdout
});

const printStars = (n) => {
    n = parseInt(n);

    for (let i = 0; i < n; i++) {
        console.log(' '.repeat(n - i - 1) + '*'.repeat(2 * i + 1));
    }

    for (let i = n - 2; i >= 0; i--) {
        console.log(' '.repeat(n - i - 1) + '*'.repeat(2 * i + 1));
    }

    readline.close();
}

readline.question('Enter a number: ', printStars);

File: task-03/Rust/SubTask3.rs
use std::io;

fn main() {
    println!("Enter a number: ");
    let mut input = String::new();
    io::stdin().read_line(&mut input).expect("Failed to read line");
    let n: usize = input.trim().parse().expect("Please enter a number");

    for i in 0..n {
        println!("{}{}", " ".repeat(n - i - 1), "*".repeat(2 * i + 1));
    }
    for i in (0..n-1).rev() {
        println!("{}{}", " ".repeat(n - i - 1), "*".repeat(2 * i + 1));
    }
}
